packages/repository/src/repositories/repository.ts

```diff
@@ -185,6 +185,14 @@ export interface EntityCrudRepository<T extends Entity, ID>
    * Promise<false>
    */
   exists(id: ID, options?: Options): Promise<boolean>;
+
+  /**
+   * Find one matching record. Same as 'find', but limited to one result.
+   * @param filter Query filter
+   * @param options Options for the operations
+   * @returns A promise of the target object or null if not found.
+   */
+  findOne(filter?: Filter<T>, options?: Options): Promise<T | null>;
```

Comment:
We should be using `undefined` instead of `null` because it works better with ES6 default parameters.